api/v1alpha1/queuedworkload_types.go

```diff
@@ -129,6 +131,10 @@ type QueuedWorkloadCondition struct {
 type QueuedWorkloadConditionType string
 
 const (
+	// QueuedWorkloadPending means that the QueuedWorkload has not been
+	// admitted by a ClusterQueue.
+	QueuedWorkloadPending QueuedWorkloadConditionType = "Pending"
```

Comment:
hmm, I don't think we want both Pending and Admitted because a workload can't be both. Use Admitted as the type with ConditionStatus=false